clusters/app.ci/supplemental-ci-images/hypershift/README.md

```diff
@@ -0,0 +1,8 @@
+# Special notes for building HyperShift images
+* `hypershift-cli` image serves both as the CLI and the operator.
+* Due to lack of forward/backward-compatibility of HyperShift, it is critical that we tag the build. For instance, once a build is done and pushed to `latest`, we shall tag it like
+```
+oc --context app.ci -n ci tag hypershift-cli:latest hypershift-cli:20230428-12e6a502bd6a7ea5434df2b83fef102b7819b413 --as system:admin
+```
+The naming convention is `YYYYMMDD-<git commit hash>`.
```

Comment:
`git commit hash` of which repo?

Reply:
the git hash of [hypershift-operator](https://github.com/openshift/hypershift/) repo

Reply:
I am lost again. Where do you check out the repo? I do not see it in this doc.